language_detector/main.py

```diff
@@ -8,19 +8,20 @@ def detect_language(text, languages=LANGUAGES):
 	max_common_words = 0
 	for language in languages:
 		common_words = language["common_words"]
-		common_words_count = getCount(text, common_words)
-		print(language["name"], common_words_count)
+		common_words_count = get_common_words_count(text, common_words)
 		if common_words_count > max_common_words:
 			max_common_words = common_words_count
 			ans = language["name"]
 	return ans
 
-def getCount(text, words):
-	ans = 0
-	lower_text = text.lower()
-	set_text = set(lower_text.split(" "))
-	for word in words:
-	    if word in set_text:
-	        ans += 1
-	return ans/(len(set_text))
+def get_common_words_count(text, words):
```

Comment:
We like the fact that you guys have accounted for normalizing the input text based on size.